pkg/services/provisioning/datasources/datasources.go

```diff
@@ -12,14 +12,6 @@ import (
 	jsoniter "github.com/json-iterator/go"
 )
 
-type Store interface {
```

Comment:
Note: I could leave it in, but I think it's cleaner when you can see that DS service is used (previously it looked like it was DS store that was called, although it wasn't the case).

Reply:
I've renamed `DataSourceService` in DS provisioning to `BaseDataSourceService`, I think that's what you meant, right? I initially removed the interface altogether and used the DS service interface directly, but it required pretty big test updates, so I decided to leave the interface in.